Advantages

julika-gulchitai opened the pull request:
An advantages section was created

julika-gulchitai committed:
section advantages

julika-gulchitai committed:
Merge branch 'main' into advantages

julika-gulchitai committed:
advantages pictures correction

defaultnastia commented on src/partials/advantages.html:
```diff
@@ -0,0 +1,68 @@
+        <img src="./img/advantages/advantages.webp" alt="advantages" 
+        loading="lazy"/>
+      </picture>
+      <button class="blueButton button">Download</button>
```
1. це має бути
`<a class="blueButton" href="https://play.google.com/store/apps/details?id=com.beachside.mergetown">
       Download
        </a>`

2. в стилі для .advantagesGigImage .blueButton треба додати z-index: 100; як мінімум на desktop щоб працював ховер

defaultnastia commented on src/partials/advantages.html:
```diff
@@ -0,0 +1,68 @@
+<section class="advatages">
```
виправ і додай класів і айді секції `<section class="advantages section" id="advantages">`
і ще одруківку advantages (не advatages)

defaultnastia commented on src/scss/layout/advantages.scss:
```diff
@@ -0,0 +1,102 @@
+.advatages {
```
коли ти додасиш клас section то відступ вже не треба, а ширину задає клас container
але треба додати вирівнювання і виправити одруківку advantages (не advatages)
`.advantages {
  margin: 0 auto;
}`

julika-gulchitai committed:
add target blank

defaultnastia reviewed (changes requested):
скрипти не працюють

julika-gulchitai committed:
fix bugs

julika-gulchitai committed:
fix mobilemenu script

julika-gulchitai commented:
All has been done. However, attention is required to hover over blueButton in styles.scss

defaultnastia committed:
updated vars and styles

- updated vars
- updated blreButton hover style not to be shown on mobile
- updated mobile color
- updated footer background color
- updated z-indexes

defaultnastia committed:
Merge branch 'main' into advantages
